### PR TITLE
MAPI-1214 add importance to legacy notification object

### PR DESCRIPTION
**Pull request by pvighi**

### CODE REVIEW

File: src/main/scala/com/gu/mobile.notifications.client/models/legacy/Notification.scala [hunk 1]
```diff
@@ -43,7 +44,8 @@ case class Notification(
     * If the client wants anything more advanced they should maintain their own data store using the notification ID
     * returned by Pushy when they deliver the notification.
     */
-  metadata: Map[String, String]
+  metadata: Map[String, String],
+  importance: Importance
```
Comment (agulowaty) [inline src/main/scala/com/gu/mobile.notifications.client/models/legacy/Notification.scala]: [minor] please remove space before `i`

File: src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala [hunk 2]
```diff
@@ -39,10 +40,11 @@ class LegacyApiClientSpec extends ApiClientSpec[LegacyApiClient] {
     target = Target(regions = Set(UK), topics = Set(Topic.NewsstandIos)),
     timeToLiveInSeconds = 10,
     payloads = MessagePayloads(Some(iosPayload), Some(androidPayload)),
-    metadata = Map("m1" -> "v1")
+    metadata = Map("m1" -> "v1"),
+    importance = Major
```
Comment (agulowaty) [inline src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala]: same as above

**Comment by agulowaty** [comment]
:+1:

File: src/main/scala/com/gu/mobile.notifications.client/models/legacy/Notification.scala (hunk 1 above)
Comment (pvighi) [inline src/main/scala/com/gu/mobile.notifications.client/models/legacy/Notification.scala]: Where is the extra space?
If you mean before importance, it is at the same indentation level as metadata and all the other fields

File: src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala (hunk 2 above)
Comment (pvighi) [inline src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala]: same response as above

Comment (agulowaty) [inline src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala]: you're right, sorry

Comment (pvighi) [inline src/test/scala/com/gu/mobile/notifications/client/LegacyApiClientSpec.scala]: No problem, thanks for reviewing the PR

**Pull request merged**